### DISCUSSION TITLE
Version info option

### DISCUSSION THREAD

**Discussion by TobiasPSP** [egzumer/uv-k5-firmware-custom#191] [Ideas]
Thanks for the awesome work!
Maybe I overlooked it, but apparently there is no way to find out the current version uploaded to a radio. Chirp and other tools display a generic text that is the same for different versions. Would it be possible to add a menu option to query the currently installed version? If there is already such a feature, could you kindly point me to it? Many thx!

**Comment by TobiasPSP** [comment]
You can disregard this post, sorry about that. I just flashed the latest version 0.19 and found that beginning with this version, the firmware version string has been changed to a meaningful text now. Many thanks, problem resolved.

**Comment by ErikS-web** [comment]
Quick problem solving ;-)
